geronimo-openapi-impl/src/main/java/org/apache/geronimo/microprofile/openapi/impl/processor/SchemaProcessor.java

```diff
@@ -99,7 +100,7 @@ public void fillSchema(
             final Supplier<org.eclipse.microprofile.openapi.models.Components>components,
             final Type rawModel,
             final org.eclipse.microprofile.openapi.models.media.Schema schema,
-            final String providedRef) {
```

Comment:
Maybe keep it final to avoid to change inputs within the method (just the overall pattern so probably saner to keep it unified IMHO)

Reply:
yeah true. Changed it back to final